### PR TITLE
model-serving/tfserving: adds tfserving use case resources

### PR DESCRIPTION
**Pull request by DnPlas**

### CODE REVIEW

File: model-serving/tfserving/README.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+```bash
+$ docker run --rm -v ${PWD}/shared/models/:/models -p 8501:8501 dlrs-serving:v0.3
+
+# For my_model v1 --> http://localhost:8501/v1/models/my_model/versions/1:predict
+# Or by its label --> http://localhost:8501/v1/models/my_model/labels/stable:predict
+# For my_model v2 --> http://localhost:8501/v1/models/my_model/versions/2:predict
+# Or by its label --> http://localhost:8501/v1/models/my_model/labels/dev:predict
+```
+
+### Moniroting
```
Comment (Meadosc) [inline model-serving/tfserving/README.md:175]: 175 lines in and this is the first typo, very nice.

File: model-serving/tfserving/README.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+$ docker build -t dlrs-serving:v0.4 -f Dockerfile.serving .
+```
+
+5. Run the container image again. You will be able to make requests to each of the models you uploaded by pointing to the model name.
+
+```bash
+$ docker run --rm -v ${PWD}/shared/models/:/models -p 8501:8501 dlrs-serving:v0.4
+```
+
+You should now be able to access the Prometheus web UI to monitor your model server. You can watch metrics by typying `:tensorflow:core` for TensorFlow core metrics or `:tensorflow:cc` in the search bar.
```
Comment (Meadosc) [inline model-serving/tfserving/README.md:208]: Typo: typying

**Review by Meadosc — APPROVED** [review]
Looks great, just needs a few typos fixed before merging.

File: model-serving/tfserving/README.md (hunk 1 above)
Comment (DnPlas) [reply model-serving/tfserving/README.md:175]: Hahaha, oh my dyslexia

**Review by unrahul — APPROVED** [review]
+2 LGTM

**Pull request merged**